Package.swift

```diff
@@ -74,7 +74,8 @@ let package = Package(
             exclude: [
                 "README.md",
                 "Tests",
-                "Source/Supporting Files"
+                "Source/Supporting Files",
+                "Swagger/CONFIG.md"
```

Comment:
Really this file should be removed. The config should live in the the rest API specs repo's README.md

Reply:
closing this - will move any config to a readme in specs repo like you mentioned